src/models/global.rs

```diff
@@ -150,20 +172,33 @@ impl Default for V4L2Standard {
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Validate)]
 #[serde(default, rename_all = "camelCase", deny_unknown_fields)]
 pub struct GrabberV4L2 {
+    pub enable: bool,
     pub device: String,
     pub input: i32,
     pub standard: V4L2Standard,
     pub width: u32,
     pub height: u32,
+    pub encoding: String,
```

Comment:
This should be an enum, like it is done for the standard field.

Reply:
True that:

https://github.com/hyperion-project/hyperion.ng/blob/master/src/hyperion-v4l2/hyperion-v4l2.cpp#L146
   |
   v
https://github.com/hyperion-project/hyperion.ng/blob/master/include/utils/PixelFormat.h#L71

```rs
#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
#[serde(rename_all = "SCREAMING_SNAKE_CASE", deny_unknown_fields)]
pub enum Pixelformat {
    NoChange,
    Yuyv,
    Uyvy,
    Bgr16,
    Bgr24,
    Rgb32,
    Bgr32,
    I420,
    Nv12,
    Mjpeg,
}

impl Default for Pixelformat {
    fn default() -> Self {
        Self::NoChange
    }
}
```

Reply:
Digging through the source to find actual config values is the hard part of this rewrite tbh, good find!